src/support/timefuncs.c

```diff
@@ -29,30 +29,31 @@ extern "C" {
 #endif
 
 #if defined(_OS_WINDOWS_)
-static double floattime(void)
+int gettimeofday(struct timeval *tv, void *tz)
 {
-    struct timeb tstruct;
-
-    ftime(&tstruct);
-    return (double)tstruct.time + (double)tstruct.millitm/1.0e3;
+    struct timeb tb;
+    ftime(&tb);
```

Comment:
It seems like we should call [_ftime64_s](https://msdn.microsoft.com/en-us/library/95e68951.aspx) which has the advantages that it (a) returns an error code and (b) uses a 64-bit seconds counter.